### PR TITLE
[OSDOCS-4014] Created an ingress Operator responsibility matrix

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
Enterprise-4.11+

Issue:
[OSDOCS-4014](https://issues.redhat.com/browse/OSDOCS-4014)

Link to docs preview:

- **[OSD](http://file.rdu.redhat.com/eponvell/OSDOCS-4014_Responsibility-Matrix/dedicated/networking/ingress-operator.html)**
- **[ROSA](http://file.rdu.redhat.com/eponvell/OSDOCS-4014_Responsibility-Matrix/rosa/networking/ingress-operator.html)**

Additional information:
This PR continues the efforts of [OSDOCS-3870](https://issues.redhat.com/browse/OSDOCS-3870). For this PR, I added the responsibility matrix for what components are covered by SRE.

### CODE REVIEW

File: modules/sd-ingress-responsibilities.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The following table details the components of the ingress Operator and if Red Hat Site Reliability Engineers (SRE) maintains this component on {product-title} clusters.
+
+.Table 1 - Ingress Operator Responsibility Chart
+
+[cols="3,2a",options="header"]
+|===
+
+|Ingress component
+|Managed by
```
Comment (dustman9000) [inline modules/sd-ingress-responsibilities.adoc:15]: Can we add an additional colum, maybe "Default Config", with all set to "Yes"?

File: modules/sd-ingress-responsibilities.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// * understanding-networking.adoc
+
+[id="sd-ingress-responsibility-matrix_{context}"]
+= {product-title} Ingress Operator configurations
+
+The following table details the components of the ingress Operator and if Red Hat Site Reliability Engineers (SRE) maintains this component on {product-title} clusters.
```
Comment (jldohmann) [inline modules/sd-ingress-responsibilities.adoc:7]: [Ingress](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress) should be capitalized:
```suggestion
The following table details the components of the Ingress Operator and if Red Hat Site Reliability Engineers (SRE) maintains this component on {product-title} clusters.
```

File: modules/sd-ingress-responsibilities.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+[cols="3,2a,2a",options="header"]
+|===
+
+|Ingress component
+|Managed by
+|Default configuration?
+
+|Scaling Ingress Controller | SRE | Yes
+
+|Ingress Operator thread Count | SRE | Yes
```
Comment (jldohmann) [inline modules/sd-ingress-responsibilities.adoc:20]:
```suggestion
|Ingress Operator thread count | SRE | Yes
```

**Review by jldohmann — COMMENTED** [review]
Mostly LGTM, some minor nits

**Pull request merged**